subsys/random/rand32_entropy_device.c

```diff
@@ -67,6 +67,13 @@ static void rand_get(u8_t *dst, size_t outlen)
 	ret = entropy_get_entropy(dev, dst, outlen);
 
 	if (unlikely(ret < 0)) {
+		/* Don't try to fill the buffer in case of
+		 * cryptographically secure random numbers, just
+		 * propagate the driver error.
+		 */
+		if (csrand)
```

Comment:
coding style requires explicit braces even for one-liners

Reply:
indeed, always forget to run uncrustify. Btw shouldn't it be checked by CI ?